docs/dev/how-to/run-demo.md

```diff
@@ -0,0 +1,40 @@
+# How to run the demo
```

Comment:
This is also intended to be autogenrated with #108

Reply:
I do not understand how.